docs/12-design-changes.md

```diff
@@ -1,26 +0,0 @@
-# Design Changes
```

Comment:
I deleted this file because I think this is not necessary anymore. Other opinions?

Reply:
I'm not sure, I think it could be valuable to keep documents about major changes. Maybe this is not the right place though.

Maybe we can generalize htis and move it to the wiki? Like a doucment: "Major architecture decisions"

Reply:
Yep, would be better imo, not sure if it is necessary though. We could discuss this in the teamcall later.

Reply:
As discussed: Will keep it deleted and maybe add information about major design changes somewhere else, e.g. wiki.